packages/next/build/index.ts

```diff
@@ -63,11 +63,17 @@ export default async function build(
     )
   }
 
+  const debug =
```

Comment:
Not really that important, but I've used this module for things like this in the past: https://github.com/sindresorhus/yn